producer/smpolicy.go

```diff
@@ -255,11 +259,18 @@ func createSMPolicyProcedure(request models.SmPolicyContextData) (
 func HandleDeleteSmPolicyContextRequest(request *httpwrapper.Request) *httpwrapper.Response {
 	logger.SMpolicylog.Infoln("handle DeleteSmPolicyContext")
 	smPolicyID := request.Params["smPolicyId"]
+	response, problems := getSmPolicyContextProcedure(smPolicyID)
+	smPolicyDnn := "UNKNOWN_DNN"
+	if problems != nil {
```

Comment:
Should it be response != nil ? 
if problem is not nil then response is nil and it would lead to crash.